### PR TITLE
Wasabee Implementation

### PR DESCRIPTION
**Pull request by TemperedFool**
Add Wasabee.json file for the bee, and add the relevant lang entries. Used the textures for the Terrestrial bee as a placeholder since they're approximately the right color. Needs proper textures from @Ridanisaurus.

### CODE REVIEW

**Comment by TemperedFool** [comment]
Added recipes to craft the various SushiGoCrafting stuff with Wasabee, Wet, and Woody combs, and wax. Took the opportunity to move all Resourceful Bees shaped recipes in base to base/recipetypes/resourcefulbees/shaped.js.

**Comment by NillerMedDild** [comment]
Should I wait before reviewing? :)

**Comment by TemperedFool** [comment]
> Should I wait before reviewing? :)

Feel free to review generally, but Rid is a hero and got me textures already. Just want to test that they actually load before I commit them.

File: config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json [hunk 1]
```diff
@@ -136,7 +136,7 @@
 
    "block.resourcefulbees.boobee_honeycomb_block": "BooBee Honeycomb Block",
    "item.resourcefulbees.boobee_honeycomb": "BooBee Honeycomb",
-   "item.resourcefulbees.boobee_bee_spawn_egg": "BooBee Spawn Egg",
+   "item.resourcefulbees.boobee_bee_spawn_egg": "BooBee Bee Spawn egg",
```
Comment (NillerMedDild) [inline config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json:139]: Are you sure about this one? :P

File: config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json [hunk 2]
```diff
@@ -306,76 +306,81 @@
...
    "item.resourcefulbees.ichor_honeycomb": "Ichor Honeycomb",
-   "item.resourcefulbees.ichor_bee_spawn_egg": "Ichor Spawn Egg",
-   "entity.resourcefulbees.ichor_bee": "Ichor Bee"
+   "item.resourcefulbees.ichor_bee_spawn_egg": "Ichor Bee Spawn egg",
+   "entity.resourcefulbees.ichor_bee": "Ichor Bee",
+   
+   "block.resourcefulbees.wasabee_honeycomb_block": "Wasabee Honeycomb Block",
+   "item.resourcefulbees.wasabee_honeycomb": "Wasabee Honeycomb",
+   "item.resourcefulbees.wasabee_bee_spawn_egg": "Wasabee Spawn egg",
+   "entity.resourcefulbees.wasabee_bee": "Wasabee Bee"
```
Comment (NillerMedDild) [inline config/resourcefulbees/resources/assets/resourcefulbees/lang/en_us.json:385]: intentional or not? 😄

**Review by NillerMedDild — APPROVED** [review]

**Comment by TemperedFool** [comment]
Should be good to go @NillerMedDild!

**Pull request merged**